include/revng/Pipeline/LLVMContainer.h

```diff
@@ -46,6 +46,9 @@ class LLVMContainer : public EnumerableContainer<LLVMContainer> {
                 std::unique_ptr<llvm::Module> M) :
     EnumerableContainer<ThisType>(*Context, Name), Module(std::move(M)) {}
 
+  LLVMContainer &cloneFrom(const LLVMContainer &Another);
+  LLVMContainer &swapWith(LLVMContainer &Another);
```

Comment:
I'd squash this commit in the first commit using it.
Same for `linkModules`.

Reply:
Squashed.